app/services/time_tracker_extension/telegram_message_generators/active_task.rb

```diff
@@ -0,0 +1,36 @@
+module TimeTrackerExtension
+  module TelegramMessageGenerators
+    class ActiveTask
+      def initialize(user)
+        @user = user
+      end
+
+      def perform
+        if active_task
+          { text: message_of_found_task, reply_markup: { inline_keyboard: [[stop_button]] } }
+        else
+          { text: message_of_unfound_task }
+        end
+      end
+
+      private
+      attr_reader :user
+
+      def stop_button
+        Telegram::Bot::Types::InlineKeyboardButton.new(text: I18n.t("telegram.stop"), callback_data: "stop_active_task:#{active_task.id}")
+      end
+
+      def active_task
+        @active_task ||= user.time_records.by_workspace(user.active_workspace_id).find_by("time_start IS NOT NULL")
+      end
+
+      def message_of_found_task
+        I18n.t("telegram.active_task", description: active_task.description, time: active_task.calculated_spent_time, project: active_task.project.name)
+      end
+
+      def message_of_unfound_task
```

Comment:
I think it's better to use "message about found task" and "... not found task"

Reply:
agree